CSVLintNppPlugin/PluginInfrastructure/Lexer.cs

```diff
@@ -11,6 +11,7 @@ internal static class ILexer
     {
         public static readonly string Name = "CSVLint\0";
         public static readonly string StatusText = "CSV Linter and validator\0";
+        static readonly int LexerID = 1976;
```

Comment:
You mean `private` instead of `internal`. Yes, I am aware that even in C++ (or for that matter any respectable OOP language), class members are by default `private`. That is there to provide encapsulation, the core principle of OOP. So, declaring class members as `public` is violating that encapsulation.

`LexerID` is declared here, and then is only accessed in the `public static int GetIdentifier(IntPtr instance)` method. So, there is no compelling reason to make it `public`. Making it `public` just because adjacent class members are so is not a good programming practice.

However, if @BdR76 decides that this should also be made `public` for this PR to be accepted, I will do so but while suppressing my very valid difference of opinion.

Reply:
Thank you @rdipardo. I will defer to you on this. I have just now pushed a commit with change.